packages/chart/src/components/LineChart.tsx

```diff
@@ -36,9 +36,63 @@ export default function LineChart({ xScale, yScale, getXAxisData, getYAxisData,
     return `${formatNumber(value, axis)}`
   }
 
+  const isNumber = value => {
+    // in debugging I saw cases where inbound was a 'number'
+    // and other times a 'string' so might as well take care of both here
+    if (typeof value === 'number') {
+      return !Number.isNaN(value)
+    }
+    if (typeof value === 'string') {
+      return value !== null && value !== '' && /\d+\.?\d*/.test(value)
+    }
+    return false // if we get here something is wrong so return false
+  }
+
+  const checkCy = cy => {
+    //console.log("cy, type ", cy, typeof cy);
+    if (isNaN(cy) || Number.isNaN(cy)) {
+      return null
+    }
+    return cy
+  }
+
+  // REMOVE bad data points from the data set
+  // Examples: NA, N/A, "1,234", "anystring"
+  // - if you dont call this on data into LineGroup below, for example
+  // then entire data series are removed because of the defined statement
+  // i.e. if a series has any bad data points the entire series wont plot
+  function cleanData(data) {
```

Comment:
Repeated fn, if we're re-using this lift this up to CdcChart

Reply:
OK sure I can do that.